### DISCUSSION TITLE
How to make post views count?

### DISCUSSION THREAD

**Discussion by spanjaan** [wintercms/wn-blog-plugin#35] [General]
How to count post views in this plugin? it is very useful for blog and news site. you have any idea? please share.

**Comment by LukeTowers** [comment]
You could use an external analytics service like [Fathom Analytics](https://usefathom.com/ref/8BVKSN) or Matomo, or you could write a plugin that extends this one (or just use the PHP code section on your blog posts details page) to track a visit every time someone requests the page. I would recommend the external analytics provider generally since doing analytics as a part of your rendering process can cause a number of issues (extra load on the server, can't employing content caching strategies, potential for abuse to balloon your stats and make them completely useless, etc)
